pages/boards/index.tsx

```diff
@@ -0,0 +1,40 @@
+import BestArticleList from "@/components/boards/BestArticleList";
+import AllArticleList from "@/components/boards/AllArticleList";
+import styles from "@/components/boards/Freeboard.module.scss";
+import { ArticleProps } from "@/types/articleTypes";
+import { getArticle } from "@/lib/articleApi";
+
+const ORDERBY = "recent";
+const PAGESIZE = 10;
+const REVALIDATE = 10;
+
+export async function getStaticProps() {
+  try {
+    const bestRes = await getArticle({ orderBy: ORDERBY, pageSize: PAGESIZE });
```

Comment:
위에서 한 번 언급한 것처럼 기본값이 설정되어 있기에, orderBy와 pageSize는 따로 넣어주지 않아도 동일하게 동작합니다. 목적이나 취향에 따라 원하는 방식으로 사용해주시면 될 것 같아요.
```suggestion
    const bestRes = await getArticle({});
```

Reply:
넵 기본값이 되어있어서 필요한 경우에만 적용하면 될 거 같습니다~!🫡